SUPPORT_POLICY.rst

```diff
@@ -0,0 +1,33 @@
+Overview
+========
+This page describes the support policy for the AWS Encryption SDK. We regularly provide the AWS Encryption SDK with updates that may contain support for new or updated APIs, new features, enhancements, bug fixes, security patches, or documentation updates. Updates may also address changes with dependencies, language runtimes, and operating systems.
+
+We recommend users to stay up-to-date with Encryption SDK releases to keep up with the latest features, security updates, and underlying dependencies. Continued use of an unsupported SDK version is not recommended and is done at the user’s discretion
+
+
+Major Version Lifecycle
+========================
+The AWS Encryption SDK follows the same major version lifecycle as the AWS SDK. For details on this lifecycle, see  `AWS SDKs and Tools Maintenance Policy`_.
+
+Version Support Matrix
+======================
+This table describes the current support status of each major version of the AWS Encryption SDK for Python. It also shows the next status each major version will transition to, and the date at which that transition will happen.
+
+.. list-table::
+    :widths: 30 50 50 50
+    :header-rows: 1
+
+    * - Major version
+      - Current status
+      - Next status
+      - Next status date
+    * - 1.x
+      - Maintenance
+      - End of Support
+      - 2022-06-30
+    * - 2.x
+      - General Availability 
+      - Maintenance
+      - 2021-06-30
```

Comment:
I don't understand this date

Reply:
I'm reading this as 2.x is transitioning from General Availability -> Maintenance *today*. Should this be `2022-06-30`?

Reply:
Online conversation, this makes sense as we move to Python 2.x depreciation.